guides/micronaut-google-cloud-http-function/micronaut-google-cloud-http-function.adoc

```diff
@@ -175,9 +178,12 @@ You can then use this variable to test the function invocation:
 [source,bash]
 ----
 curl -w "\n" $YOUR_HTTP_TRIGGER_URL/micronautguide
-Example Response
 ----
 
+```
```

Comment:
I am not sure that this is proper asciidoc. I think they are Github fenced code blocks